SamplesDashboard/SamplesDashboard/Models/YamlHeader.cs

```diff
@@ -10,9 +16,129 @@ public class YamlHeaderExtensions
     }
     public class YamlHeader
     {
+        private static readonly string GitHubRawContent = "https://raw.githubusercontent.com";
```

Comment:
Can this be stored in config?

Reply:
Sure!

Reply:
Actually, it could, but it will sort of complicate things :D. The YamlHeader class doesn't have access to the `IConfiguration` instance. So I'd have to pull it out in the service and pass it as a parameter, or reconfigure the YAML stuff into it's own service. I don't anticipate that GitHub will change their raw content domain anytime soon.